### PR TITLE
roof removal: Add Barrows Tunnel override

### PR DESCRIPTION
**Pull request by hansjm10**
Add Barrow Tunnel to the roof removal override config.

Fixes [#15361](https://github.com/runelite/runelite/issues/15361)

### CODE REVIEW

**Review by Hydrox6 — CHANGES_REQUESTED** [review]
Just the tiniest nitpick to keep the file clean-ish. Otherwise, the override looks fine :+1:

**Comment by hansjm10** [comment]
> 

My bad didn't even realize the whitespace was off! Should be fixed now.

File: runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc [hunk 1]
```diff
@@ -3754,5 +3754,15 @@
       "z1": 2,
       "z2": 2
     }
+  ],
+   "14231": [ //Barrows Tunnels
```
Comment (Hydrox6) [inline runelite-client/src/main/resources/net/runelite/client/plugins/roofremoval/overrides.jsonc:3758]:
```suggestion
   "14231": [ // Barrows Underground
```

**Review by Hydrox6 — CHANGES_REQUESTED** [review]
I meant to leave this review comment before, but didn't click `Start Review`. oops

**Review by Hydrox6 — APPROVED** [review]

**Pull request merged**